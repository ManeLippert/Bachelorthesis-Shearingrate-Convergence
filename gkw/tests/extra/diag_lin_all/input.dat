!Waltz linear test case

nmp=16,1

 &CONTROL
 silent = .false.,
 order_of_the_scheme = 'fourth_order'
 fac_dtim_est=5.0
 READ_FILE  = .false. !Any easy way to set this to true in test cases?
 NON_LINEAR = .false.
 zonal_adiabatic = .true.,
 METHOD = 'EXP',
 METH   = 2, 
 DTIM   = 0.001 
 NTIME = 10
 NAVERAGE = 20
 nlapar = .true.
 nlbpar = .true.
 collisions = .true.
 disp_par = 1.0
 disp_vp  = 0.05
 lverbose = .true.
 gamatol = 1.e-5
 /
 &GRIDSIZE
 n_procs_s = 2
 n_procs_vpar = 2
 n_procs_mu = 2
 n_procs_sp = 2
 NX = 1
 N_s_grid = 50
 N_mu_grid = 8,
 nperiod = 2
 N_vpar_grid = 16, 
 NMOD = 1
 number_of_species = 2 ! Number of species. Note: do not count the adiabatic species  
 /
 &MODE
 CHIN    = 0., 
 KTHRHO  = 0.42426,
 mode_box = .false.
 krhomax = 1.485,
 ikxspace = 1, 
 /
 &GEOM 
 SHAT = 1.0,
 Q    = 2., 
 EPS  = 0.166
 /
 &SPCGENERAL
 beta = 0.010000e+00
 adiabatic_electrons = .false.
 amp_init = 0.5e-2
 finit='cosine'
 /
 &SPECIES 
 MASS  = 1., 
 Z     = 1., 
 TEMP  = 1., 
 dens  = 1.,
 rlt   = 9.,
 rln   = 3., 
 uprim = 0.2,
 / 
 &SPECIES 
 MASS  = 2.72e-4, 
 Z     = -1.0, 
 TEMP  = 1. 	
 dens  = 1.0,
 rlt   = 9.,
 rln   = 3., 
 uprim = 0.2,
 / 
 &ROTATION
 VCOR = 0.13
 cf_trap = T,
 cf_drift = T
 / 
 &COLLISIONS
 rref = 1.3			!reference major radius used in collision operator
 tref = 1.4			!reference temperature in units of kev used for the collision operator
 nref = 1.5			!reference density in units 10^19 m^-3 used for the collision operator
 pitch_angle = .true.           !These three terms swith on various terms in the collision operator, this one is pitch angle scattering
 en_scatter =  .true.           !Energy scattering term
 friction_coll = .true.
 mom_conservation = .true.
 mass_conserve = .true.
 freq_override = .true.
 coll_freq = 0.0001
 /
!---------------------------------------------------------------------------------------------------------------------
 &DIAGNOSTIC  !> Optional namelist read in diagnostics.F90 to switch individual diagnostics on / off
              !< This is for the diagnostic switches; some switches from elsewhere will be moved here
 NFLUSH_TS = 2         ! The number of large time steps between flushing fluxes,time etc. to file  (default = 10)
 LPFLUX=T              ! Enable output of pflux in fluxes - time diagnostic                        (default = T)
 LEFLUX=T              ! Enable output of eflux in fluxes - time diagnostic                        (default = T)
 LVFLUX=T              ! Enable output of vflux in fluxes - time diagnostic                        (default = T)
 LFLUXES_SPECTRA=T     ! Enable output of ES fluxes spectra - time diagnostic                      (default = T)
 LFLUXES_EM_SPECTRA=T  ! Enable output of EM (apar) fluxes spectra - time diagnostic               (default = F)
 LFLUXES_DETAIL=T      ! writes the fluxes as a function of s, mu and vpar                         (default = F)
 LPARALLEL_OUTPUT =T   ! writes the output of the moments along the magnetic field                 (default = T)
 LROTATE_PARALLEL=T    ! normalizes the parallel output such that Re[phi(s=0)] = 1.                (default = T)
 LPARALLEL_PHI=T       ! Enable time output of electrostatic potential along field line            (default = T)
 LPARALLEL_APAR=T      ! Enable time output of A_// magnetic  potential along field line           (default = F)
 LMPI_BROKEN_IO=T      ! If MPI_IO fully supports derived datatypes, use F for faster file write   (default = T)
! lisl_follow=F         ! Makes the X-Y diagnostics follow the o-point of rotating magnetic island  (default = T)
! phi3d = T       ! 3d electrostatic potential in real space (binary)                         (default = F)
! spc3d = T       ! 3d electrostatic potential spectrum (binary)                              (default = F)
! apa3d = T       ! 3d electromagentic potential in real space (binary)                       (default = F)
! apc3d = T       ! 3d electromagentic potential spectrum (binary)                            (default = F)
! bpa3d = T       ! 3d compressional magnetic field in real space (binary)                    (default = F)
! bpc3d = T       ! 3d compressional magnetic field spectrum (binary)                         (default = F)
! den3d = T.       ! 3d density for all species in real space (binary) - D3d0*_* files         (default = F)
! ene3d = T.       ! 3d energy output for all species in real space (binary) - E3d0*_* files   (default = F)
! zonal_scale_3d = 1.0  ! Factor by which to scale the zonal flows in the 3D output                 (default = 1.0)
! lphi_diagnostics = T   !Meta switch for 2-D diaganostics for nonlinear or mode_box runs      (default = T)
! xy_phi = T        ! Electrostatic potential in perpendicular plane at LFS midplane            (default = T)
! xy_apar = T       ! Parallel vector potential in perpendicular plane at LFS midplane          (default = F)
! xy_bpar = T       ! Compressional magnetic field in perpendicular plane at LFS midplane       (default = F)
! xy_dens = T      ! Per species densities in perpendicular plane at LFS midplane              (default = F)
! xy_temp = T      ! Per species temperatures in perpendicular plane at LFS midplane           (default = F)
! xy_current = T   ! Per species parallel currents in perp plane at LFS midplane               (default = F)
! xy_fluxes = T    ! Per species radial electrostatic heat fluxes in perp plane at LFS mp      (default = F)
! xy_fluxes_em = T ! Per species radial electromagnetic heat fluxes in perp plane at LFS mp    (default = F)
! xy_fluxes_bpar = T     ! Per species radial compressional heat fluxes in perp plane at LFS mp      (default = F)
! xy_estep = T           ! Logical that enables XY output at every large time step, otherwise output 
                        ! is just at the end of the run                                             (default = F)
 xy_slice_ipar = 2      ! The point on the s grid that the XY outputs are output from (usually LFS) (default = n_s_grid/2+1)
 LWRITE_OUTPUT1  = T     ! Meta switch for all per timestep output (fluxes, time traces, 2D)   (default = T)
 LCALC_FLUXES    = T     ! Calculate fluxes                                                    (default = T)
 LCALC_FREQ      = T    ! Calculate the mode frequencies                                      (default = T)
 LFINAL_OUTPUT   = T     ! Write final output                                                  (default = T)
 screen_output = T       ! Write per timestep data to screen  (fluxes, timestep, growth rates) (default = T) 
 LAMPLITUDES     = T,         ! Amplitudes diagnostic per mode (kx, ky 2D)                          (default = F)
 LGROWTH_RATES   = T          ! Output growth rates diagnostic per toriodial mode (sum over kx)     (default = F)
 LFREQUENCIES    = T          ! Output frequencies diagnostic per toroidal mode (kx=0 only)         (default = F)
 PSI_VELSPACE    = 0.,0.,0.,  ! List psi coordinates of points for velocity space diagnostic        (list max 32)
 ZETA_VELSPACE   = 0.,0.,0.,  ! List zeta coordinates of points for  velocity space diagnostic      (list max 32)
 NPOINTSVEL      = 0          ! Number of points for velocity diagnostic (if xy_dens)   (max = 32)  (default = 0)
 /


