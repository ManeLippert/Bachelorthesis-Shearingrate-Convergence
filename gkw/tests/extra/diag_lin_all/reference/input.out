 ! Output data generated by GKW version 2188M   
 !  2 June 2011   4:54:26.088 PM
 !  executable name: gkw_qa0-DP_2188M.x              
 !  compiled with $(FC) 
 !/afs/@cell/common/soft/intel/impi/4.0.0/bin64/mpiifort                                                                          
 !Run on            1 processor(s)
 !If the input file generated warnings:
 !This file is a clean version that should not generate warnings
 !The following are the input variables as used by the code:
 &CONTROL
 COLLISIONS      = T,
 DISP_PAR        =   1.00000000000000     ,
 DISP_VP =  5.000000000000000E-002,
 DISP_X  =  0.000000000000000E+000,
 DISP_Y  =  0.000000000000000E+000,
 DTIM    =  1.000000000000000E-003,
 IRUN    =           0,
 LVERBOSE        = T,
 MATRIX_FORMAT   = complex                         ,
 MAX_SEC =          -1,
 METH    =           2,
 METHOD  = EXP                             ,
 NAVERAGE        =          20,
 NEOCLASSICS     = F,
 NLAPAR  = T,
 NLPHI   = T,
 NON_LINEAR      = F,
 NL_DTIM_EST     = F,
 NORMALIZED      = T,
 NTIME   =          10,
 ORDER_OF_THE_SCHEME     = fourth_order                    ,
 PARALLEL_BOUNDARY_CONDITIONS    = zero                            ,
 READ_FILE       = F,
 SILENT  = F,
 TESTING = F,
 VP_TRAP =           0,
 UNIFORM_MU_GRID = F,
 LFLAPV  = F,
 ZONAL_ADIABATIC = T,
 DT_MIN  =  0.000000000000000E+000,
 NDUMP_TS        =           0,
 MAX_SECONDS     =  -1.00000000000000     ,
 AUTO_RESTART    = F,
 LTRAPPING_ARAKAWA       = F,
 NLBPAR  = T,
 RESTART_FILE_VERSION    =           2,
 GAMATOL =  1.000000000000000E-005,
 AMP_IMOD        =  0.000000000000000E+000,
 IMOD_INIT       =        -999,
 AMP_ZON =  0.000000000000000E+000,
 MODE_PERSIST    = F,
 NORMALIZE_PER_TOROIDAL_MODE     = F,
 USE_NON_ADIA    = F
 /
 &GRIDSIZE
 NX      =           1,
 NPERIOD =           2,
 N_VPAR_GRID     =          16,
 NMOD    =           1,
 NUMBER_OF_SPECIES       =           2,
 N_MU_GRID       =           8,
 N_TRAPPED       =           0,
 N_S_GRID        =          50,
 N_PROCS_S       =           2,
 N_PROCS_SP      =           2,
 N_PROCS_VPAR    =           2,
 N_PROCS_MU      =           2,
 NON_BLOCKING_VPAR       = T,
 VPMAX   =   3.00000000000000     ,
 MUMAX   =   4.50000000000000     
 /
 &DIAGNOSTIC
 LPFLUX  = T,
 LEFLUX  = T,
 LVFLUX  = T,
 LFLUXES_SPECTRA = T,
 LFLUXES_EM_SPECTRA      = T,
 LPARALLEL_PHI   = T,
 LPARALLEL_APAR  = T,
 LFLUXES_DETAIL  = T,
 NFLUSH_TS       =           2,
 LWRITE_OUTPUT1  = T,
 LCALC_FLUXES    = T,
 LCALC_FREQ      = T,
 LFINAL_OUTPUT   = T,
 LISL_FOLLOW     = T,
 SCREEN_OUTPUT   = T,
 LMPI_BROKEN_IO  = T,
 LPHI_DIAGNOSTICS        = T,
 XY_FLUXES       = F,
 XY_DENS = F,
 XY_TEMP = F,
 XY_CURRENT      = F,
 XY_PHI  = F,
 XY_APAR = F,
 XY_BPAR = F,
 XY_FLUXES_EM    = F,
 XY_ESTEP        = F,
 XY_SLICE_IPAR   =           2,
 XY_FLUXES_BPAR  = F,
 LAMPLITUDES     = T,
 LGROWTH_RATES   = T,
 LFREQUENCIES    = T,
 SPC3D   = F,
 PHI3D   = F,
 DEN3D   = F,
 APA3D   = F,
 APC3D   = F,
 ENE3D   = F,
 BPA3D   = F,
 BPC3D   = F,
 ZONAL_SCALE_3D  =   1.00000000000000     ,
 LPARALLEL_OUTPUT        = T,
 LROTATE_PARALLEL        = T,
 PSI_VELSPACE    = 32*0.000000000000000E+000  ,
 ZETA_VELSPACE   = 32*0.000000000000000E+000  ,
 NPOINTSVEL      =           0
 /
 &GEOM
 SHAT    =   1.00000000000000     ,
 Q       =   2.00000000000000     ,
 EPS     =  0.166000000000000     ,
 EPS_TYPE        =           1,
 GEOM_TYPE       = s-alpha                         ,
 EQFILE  = not_needed_for_s-alpha_option                                                                                             
                                                           ,
 SIGNB   =           1,
 SIGNJ   =           1,
 R0_LOC  = LFS                             
 /
 &MODE
 KTHRHO  =  0.424260000000000     , 511*0.000000000000000E+000  ,
 CHIN    =  0.000000000000000E+000,
 MODE_BOX        = F,
 KRHOMAX =  0.000000000000000E+000,
 IKXSPACE        =           0
 /
 &ROTATION
 VCOR    =  0.130000000000000     ,
 SHEAR_RATE      =  0.000000000000000E+000,
 SHEAR_PROFILE   = none                            ,
 TOROIDAL_SHEAR  = none                            ,
 CORIOLIS        = T,
 CF_TRAP = T,
 CF_DRIFT        = T,
 T_SHEAR_BEGIN   =  0.000000000000000E+000,
 CF_UPPHI        = T,
 CF_UPSRC        = T
 /
 &SPCGENERAL
 BETA    =  0.000000000000000E+000,
 BETA_REF        =  1.000000000000000E-002,
 BETA_TYPE       = ref                             ,
 BETAPRIME_REF   =  0.000000000000000E+000,
 BETAPRIME_TYPE  = ref                             ,
 ADIABATIC_ELECTRONS     = F,
 AMP_INIT        =  5.000000000000000E-003,
 FINIT   = cosine                          ,
 WSTAR   =  0.000000000000000E+000,
 LS      =   2.00000000000000     ,
 ISL_ROT_FREQ    =  0.000000000000000E+000,
 TEARINGMODE     = F,
 ISL_MODE        =           2,
 TEAR_ZERO_EPAR  = F
 /
 &SPECIES
 MASS    =   1.00000000000000     ,
 Z       =   1.00000000000000     ,
 TEMP    =   1.00000000000000     ,
 DENS    =   1.00000000000000     ,
 RLT     =   9.00000000000000     ,
 RLN     =   3.00000000000000     ,
 UPRIM   =  0.200000000000000     ,
 BACKGROUND      = maxwell,
 PARAM   =  0.000000000000000E+000,
 NUF     =   1.00000000000000     
 /
 &SPECIES
 MASS    =  2.720000000000000E-004,
 Z       =  -1.00000000000000     ,
 TEMP    =   1.00000000000000     ,
 DENS    =   1.00000000000000     ,
 RLT     =   9.00000000000000     ,
 RLN     =   3.00000000000000     ,
 UPRIM   =  0.200000000000000     ,
 BACKGROUND      = maxwell,
 PARAM   =  0.000000000000000E+000,
 NUF     =   1.00000000000000     
 /
 &LINEAR_TERM_SWITCHES
 LVPAR_GRAD_DF   = T,
 LVDGRADF        = T,
 LTRAPDF = T,
 LVE_GRAD_FM     = T,
 LVD_GRAD_PHI_FM = T,
 LVPGRPHI        = T,
 LPOISSON        = T,
 LAMPERE = T,
 LG2F_CORRECTION = T,
 LPOISSON_ZF     = T,
 LNEOCLASSICAL   = T,
 LBPAR   = T
 /
 &COLLISIONS
 NREF    =   1.50000000000000     ,
 TREF    =   1.40000000000000     ,
 RREF    =  0.000000000000000E+000,
 PITCH_ANGLE     = T,
 EN_SCATTER      = T,
 FRICTION_COLL   = T,
 MOM_CONSERVATION        = T,
 MASS_CONSERVE   = T,
 FREQ_OVERRIDE   = T,
 COLL_FREQ       =  1.000000000000000E-004,
 SELFCOLLCON     = T
 /
